helpers/Authoring.php

```diff
@@ -155,16 +156,16 @@ public static function sanitizeQtiXml($qti)
     {
         $doc = self::loadQtiXml($qti);
 
-        $xpath = new \DOMXpath($doc);
+        $xpath = new DOMXpath($doc);
 
-        foreach ($xpath->query("//*[local-name() = 'itemBody']//*[@style]") as $elementWithStyle) {
-            $elementWithStyle->removeAttribute('style');
-        }
+//        foreach ($xpath->query("//*[local-name() = 'itemBody']//*[@style]") as $elementWithStyle) {
```

Comment:
Good catch @wazelin . IF you feel we may have trouble disabling it, since it is enabled for 10y, we may add a FF for that and enable it only for the client that wants to use this:

```
if ($featureFlagChecker->isEnabled('FEATURE_FLAG_ALLOW_QTI_STYLE_ATTR')) {
...
}
```

Reply:
That restriction is very superficial: it only removes the attribute on import. We support it throughout the rest of the flow: export, publication, compilation, runtime.

Both Terre UI and Solar UI work fine.

That does not make our item bank QTI incompliant – we still support the strict QTI items.